pkg/get/tools.go

```diff
@@ -3687,5 +3687,30 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 				https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}_{{.VersionNumber}}_{{$osStr}}_{{$arch}}.{{$extStr}}`,
 		})
 
+	tools = append(tools,
+		Tool{
+			Owner:       "charmbracelet",
+			Repo:        "vhs",
+			Name:        "vhs",
+			Description: "CLI for recording demos",
+			URLTemplate: `
+					{{$arch := .Arch}}
+					{{ if (eq .Arch "x86_64") -}}
+					{{$arch = "x86_64"}}
+					{{- else if eq .Arch "aarch64" -}}
+					{{$arch = "arm64"}}
+					{{- end -}}
+
+					{{$osStr := ""}}
+					{{- if eq .OS "darwin" -}}
+					https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}_Darwin_{{$arch}}.tar.gz
```

Comment:
I think you may be able to omit the prefix of https://github.com/{{.Owner}}/{{.Repo}}/releases/download/

Or just have it on the last line so it's not duplicated.

Please look at other similar CLIs.

Reply:
I have updated, PTAL.
The URL format is different in the case of Darwin and Linux/Windows therefore need 2 separate cases.

Please inform if any issues.